plugins/source/aws/scripts/policy_docs/main.go

```diff
@@ -113,26 +128,39 @@ func extractQueries(prefix, sqlPath string) ([]Query, error) {
 	}
 	content := string(b)
 	var queries []Query
-	tableMatches := reTable.FindAllStringSubmatch(content, -1)
-	reIsPolicyQuery := regexp.MustCompile(`(?i)insert\s+into\s+` + prefix + `_policy_results`)
-	isPolicyQuery := reIsPolicyQuery.MatchString(content)
-	if isPolicyQuery && len(tableMatches) > 0 {
-		q := Query{}
+	q := Query{}
+	if tableMatches := reTable.FindAllStringSubmatch(content, -1); len(tableMatches) > 0 {
 		for _, m := range tableMatches {
-			if !strings.HasPrefix(m[1], prefix) {
-				continue
+			switch {
+			case strings.HasPrefix(m[1], prefix):
+				q.Tables = append(q.Tables, m[1])
+				q.Path = sqlPath
+			case strings.HasPrefix(m[1], "view_"+prefix):
```

Comment:
This only works if names of views are clearly marked with `view_{prefix}`. We'd have to do a multipass if we don't want this requirement (which currently GCP doesn't fit)